doc/source/contributing_code.rst

```diff
@@ -217,11 +217,13 @@ First we need to install dependencies. You'll need a mix of conda and pip instal
             griddataformats gsd hypothesis "joblib>=0.12" \
             matplotlib mmtf-python mock netcdf4 networkx \
             "numpy>=1.18.0" psutil pytest scikit-learn scipy \
```

Comment:
we probably should update the numpyu minimum pin too

Reply:
yes, I just wanted to keep the commit focused

Reply:
will update to whatever I find in pyproject.toml

Reply:
If you want we can just merge this and I can just do a cleanup afterwards. There's probably a bunc of stuff in the page that needs changing.

Reply:
Just saw your comment _after_ I did a bunch of things. If you think it's useful, merge and then we can update later.

Reply:
I think you've done more or less all the changes I think needed doing. I don't think we need two PRs. (it was mostly a "if you're kinda busy we can push it to the next PR")